power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/input.hpp

```diff
@@ -215,6 +215,21 @@ struct ThreeWindingTransformerInput {
     operator Branch3Input const&() const { return reinterpret_cast<Branch3Input const&>(*this); }
 };
 
+struct TransformerTapRegulatorInput {
+    ID id;  // ID of the object
+    ID transformer_id;  // ID the of transformer
+    ControlSide control_side;  //control side of the (three winding) transformer
+    double u_set;  // voltage setpoint
+    double u_band;  // voltage bandwidth
+    bool enabled;  //regulator enabled
```

Comment:
is this better than `status`?

Reply:
And we do not use `bool` in the attribute, but `IntS`.